File: Java/11-jun-junit/src/Mathematics.java

public class Mathematics {
	int no1;
	int no2;
	int result;
	
	public Mathematics() {}

	public Mathematics(int no1, int no2, int result) {
		super();
		this.no1 = no1;
		this.no2 = no2;
		this.result = result;
	}

	public int getNo1() {
		return no1;
	}

	public void setNo1(int no1) {
		this.no1 = no1;
	}

	public int getNo2() {
		return no2;
	}

	public void setNo2(int no2) {
		this.no2 = no2;
	}

	public int getResult() {
		return result;
	}

	public void setResult(int result) {
		this.result = result;
	}
	
	public void sum()
	{
		result=no1+no2;
	}
	
	public void difference()
	{
		result=no1-no2;
	}
}

File: reactjs/uday/src/components/Mathematics.js
import { useState } from "react";

function Mathematics()
{
    const [no1, setNo1]=useState(0);
    const [no2, setNo2]=useState(0);
    const [sum, setSum]=useState(0);

    function fnAdd()
    {
        var total=parseInt(no1)+parseInt(no2);
        setSum(total);
    }

    function fnUpdate(event)
    {
        var source=event.target.id;
        switch(source)
        {
            case "no1":
                setNo1(event.target.value);
                break;
            case "no2":
                setNo2(event.target.value);
                break;
        }
        console.log("Now the state is:");
        console.log(no1);
        console.log(no2);
    }
    return <div>
        {no1}:{no2}<br/>
        Sum={sum}<br/>
        Number 1:<input type="number" id="no1" className="form-control" onChange={fnUpdate} />
        Number 2:<input type="number" id="no2" className="form-control" onChange={fnUpdate} /><br/>
        <input type="button" value="Add" className="btn btn-info" onClick={fnAdd} />
    </div>
}
export default Mathematics;